docs/examples/ipympl_example.rst

```diff
@@ -0,0 +1,127 @@
+==============
+Ipympl Example
+==============
+
+Thebe can display output from ipympl_, which enables interactivity with matplotlib_.
+
+.. _ipympl: https://github.com/matplotlib/ipympl 
+.. _matplotlib: https://matplotlib.org/ 
+
+Setup
+=====
+
+Be sure to load require.js and Font Awesome 4 before any of your thebe activation code so that the
+matplotlib widgets can function. 
+
+.. code-block:: html
+
+   <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css" integrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR8liPYU+WymTZP1lmY9G6Oc7HlZv156XqnsgNUzTyMefFTcsFH/tnJE/+xBg==" crossorigin="anonymous" />
+   <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
+
+
+Configure thebe and load it:
+
+.. code-block:: html
+
+   <script type="text/x-thebe-config">
+     {
+       requestKernel: true,
+       binderOptions: {
+         repo: "matplotlib/ipympl",
+         ref: "0.5.8",
+         repoProvider: "github",
+       },
+     }
+   </script>
+   <script src="https://unpkg.com/thebelab@latest/lib/index.js"></script>
+
+Create a button to activate thebe:
+
+.. code-block:: html
+
+   <button id="activateButton" style="width: 120px; height: 40px; font-size: 1.5em;">
+     Activate
+   </button>
+   <script>
+   var bootstrapThebe = function() {
+       thebelab.bootstrap();
+   }
+   document.querySelector("#activateButton").addEventListener('click', bootstrapThebe)
+   </script>
+
+Now add  between these HTML tags:
+
+.. code-block:: html
+
+   <pre data-executable="true" data-language="python">
+   %matplotlib widget
+
+   </pre>
+
+Examples
+========
+
+Using ipympl, you can display a variety of interactive plots.
+
+Press the "Activate" button below to connect to a Jupyter server:
+
+.. raw:: html
+
+   <script src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.4/require.min.js"></script>
+
+   <button id="activateButton" style="width: 120px; height: 40px; font-size: 1.5em;">
+     Activate
+   </button>
+   <script>
+   var bootstrapThebe = function() {
+       thebelab.bootstrap();
+   }
+   document.querySelector("#activateButton").addEventListener('click', bootstrapThebe)
+   </script>
+
+.. raw:: html
+
+   <script type="text/x-thebe-config">
+     {
+       requestKernel: true,
+       binderOptions: {
+         repo: "matplotlib/ipympl",
+         ref: "0.5.8",
+         repoProvider: "github",
+       },
+     }
+   </script>
+   <script src="https://unpkg.com/thebelab@latest/lib/index.js"></script>
+

```

Comment:
I think you should just add this line here:

```
.. warning: This cell is only required in the thebe documentation due to the way that ipympl's binder container is setup. Do not add or execute it when working with ipympl yourself.

.. raw:: html
   <pre data-executable="true" data-language="python">
    rm -r ipympl
   </pre>

```

Reply:
Just added this cell block in!